README.md

```diff
@@ -12,7 +12,7 @@
 <!-- x-hide-in-docs-end -->
 <!-- The 'github-badges' class is used in the docs -->
 <p align="center" class="github-badges">
-  <a href="https://github.com/open-feature/spec/tree/v0.7.0">
+  <a href="https://github.com/open-feature/spec/releases/tag/v0.5.2">
```

Comment:
This SDK isn't quite yet `0.6.0` compliant yet, though it's somewhat close. To me it seems `0.5.2` compliant.

Reply:
I think the main missing features are hooks and events right? Am I missing anything?

Reply:
Oh, and logging...

Reply:
I think that's the bulk of it.

FYI we had a lot of interest at KubeCon, and the Rust SDK came up a few times, so we may have some extra help incoming :crossed_fingers: